src/rdo.rs

```diff
@@ -422,6 +425,8 @@ fn compute_tx_distortion<T: Pixel>(
     (tx_dist as f64 * bias) as u64
```

Comment:
I think 'tx_dist' here has been already scaled by fi.dist_scale[0], when it is passed from encode_block_post_cdef()?